gapic/schema/wrappers.py

```diff
@@ -1830,13 +1830,35 @@ def ident(self) -> metadata.Address:
         """Return the identifier data to be used in templates."""
         return self.meta.address
 
+    def _validate_paged_field_size_type(page_field_size):
```

Comment:
```suggestion
    def _validate_paged_field_size_type(self, page_field_size):
```

Reply:
this should help the unit tests! I have absolutely been burned by this before

Reply:
@leahecole 

Resolved.